Airports/EGNM/RMA.txt

```diff
@@ -0,0 +1,4 @@
+POL - LBA Line   	 	  S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
+				          N053.46.37.004 W001.58.21.606 N053.53.48.000 W001.31.00.000
+Leeds Delegation Line	  S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
+						  N053.49.05.405 W002.01.15.038 N053.43.58.344 W001.22.49.457 ; POL - GOLES 5NM north
```

Comment:
```suggestion
POL - LBA Line              S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
                            N053.46.37.004 W001.58.21.606 N053.53.48.000 W001.31.00.000
Leeds Delegation Line       S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
                            N053.49.05.405 W002.01.15.038 N053.43.58.344 W001.22.49.457 ; POL - GOLES 5NM north
```

We tend to use spaces rather than tabs as convention - you may need to turn off an option in ya text editor to convert whitespace to tabs